app/models/bulkrax/pending_relationship.rb

```diff
@@ -3,5 +3,9 @@
 module Bulkrax
   class PendingRelationship < ApplicationRecord
     belongs_to :importer_run
+
+    # Ideally we wouldn't have a column named "order", as it is a reserved SQL term.  However, if we
+    # quote the column, all is well...for the application.
+    scope :ordered, -> { order("#{quoted_table_name}.#{connection.quote_column_name('order')}") }
```

Comment:
@jeremyf what do you mean by "for the application"?

Reply:
In this case the application logic should work.  But querying via SQL command would require quoting (whereas any non-reserved column names might not).